web/src/app/WebApp/Installers/WordPress/WordPressSetup.php

```diff
@@ -79,11 +79,11 @@ public function install(array $options = null) {
 			$result,
 		);
 		foreach ($result->raw as $line_num => $line) {
-			if ('$table_prefix =' === substr($line, 0, 15)) {
+			if (str_starts_with($line, '$table_prefix =')) {
 				$result->raw[$line_num] =
-					'$table_prefix = \'' .
-					addcslashes(Util::generate_string(5, false) . "_", "\\'") .
-					"';\r\n";
+					'$table_prefix = ' .
+					var_export("wp_" . Util::generate_string(5, false) . "_", true) .
```

Comment:
Nitpick: We could clean this up by using a bit more proper function for string formatting:

```
$result->raw[$line_num] = sprintf(
    '$table_prefix = "wp_%s_";' . "\r\n",
    '234dfgdfgd',
);
```

Reply:
i'd rather use var_export, it is *_the_* correct way to export strings to php source code.
For example, if generate_string was to return 
```
foo'bar
```
var_export would know that the ' needs escaping and return
```
'foo\'bar'
```
while sprintf would not.

Could change it to
```
$result->raw[$line_num] = sprintf(
    '$table_prefix = %s;' . "\r\n",
    var_export("wp_".Util::generate_string(5, false)."_",true),
);
```

Reply:
Okay, the above sample given by you would indeed be more cleared.

Reply:
@rjd22 code updated